content/trust-management/trust-authority.md

```diff
@@ -302,6 +302,22 @@ not** include the compromised key.
 }
 ```
 
+## Compromission of cryptographic private keys
+
+In the event of a trust authority having one or more of its cryptographic
+private keys compromised, it **must** get in touch with every trust authority
+certifying it as trustworthy and follow the same instructions as if it was a
+source ([described here]({{<relref
+"/information-distribution/source#compromission-of-cryptographic-private-keys">}})).
```

Comment:
this link is probably wrong since the target title has changed

Reply:
Good point.